x-pack/legacy/plugins/code/server/distributed/code_services.test.ts

```diff
@@ -80,13 +80,13 @@ test('non-code-node could send request to code-node', async () => {
     baseUrl: string,
     path: string,
     payload: RequestPayload,
-    originRequest: Request
+    originRequest: KibanaRequest
   ) => {
     expect(baseUrl).toBe(codeNodeUrl);
     const response = await hapiServer.inject({
       method: 'POST',
       url: path,
-      headers: originRequest.headers,
+      headers: originRequest.headers as any,
```

Comment:
Please do let us know when the change happens. There is much more context here of why we need to forward the headers. Let me add a comment here just in case, but this is not something we should handle in this PR.

Reply:
Btw, this is for the test only.